app/screens/bottom_sheet/index.tsx

```diff
@@ -69,16 +92,30 @@ const BottomSheet = ({initialSnapIndex = 0, renderContent, snapPoints = ['90%',
             style={{
                 backgroundColor: theme.centerChannelBg,
                 opacity: 1,
-                padding: 16,
+                paddingHorizontal: 16,
+                paddingBottom: isTablet ? 20 : 16,
+                paddingTop: isTablet ? 0 : 16,
                 height: '100%',
-                width: Math.min(dimensions.width, 450),
+                width: isTablet ? '100%' : Math.min(dimensions.width, 450),
                 alignSelf: 'center',
             }}
         >
             {renderContent()}
         </View>
     );
 
+    if (isTablet) {
+        const styles = getStyleSheet(theme);
+        return (
+            <>
+                <View style={styles.separator}/>
+                <View style={{flex: 1, backgroundColor: 'red'}}>
```

Comment:
1. This can be placed into the styles object `{flex: 1, backgroundColor: 'red'}`
2. Is the `red` color used for testing?

Reply:
Lol yes